pkg/k8sutil/k8sutil.go

```diff
@@ -752,6 +761,12 @@ func (k *K8sutil) CreateClientMasterDeployment(deploymentType, baseImage string,
 			},
 		}
 
+		// disable secrets volume mount when ssl is not enabled
```

Comment:
I think the idea is you won't need the volume when TLS is disabled. You could add the volumes to the object after it's created if needed. Meaning, if TLS enabled, then add the volume, but by default it won't be there.